src/components/IntegrationTest/FormikParamsField.tsx

```diff
@@ -0,0 +1,259 @@
+                    {Array.isArray(parameters) &&
+                      parameters.length > 0 &&
+                      parameters.map((p, i) => {
+                        return (
+                          <DataListItem
+                            key={`parameter-${i}`}
+                            isExpanded={expanded[i]}
+                            data-test={`its-param-${i + 1}`}
+                          >
+                            <DataListItemRow className="pf-v5-u-pl-0">
+                              <DataListToggle
+                                id={p.name}
+                                data-testid={`${p.name}-toggle`}
+                                onClick={() => toggleExpandedState(i)}
+                                isExpanded={expanded[i]}
+                                data-test={`expand-param-${i + 1}`}
+                                className="pf-v5-u-mr-0"
+                              />
+                              <DataListItemCells
+                                dataListCells={[
+                                  <DataListCell key="param-title" width={5}>
+                                    <TextContent>{`Parameter${i + 1}`}</TextContent>
+                                  </DataListCell>,
+
+                                  <DataListCell key="remove-param-button" width={3}>
+                                    <Button
+                                      isInline
+                                      type="button"
+                                      variant="link"
+                                      data-test={`remove-param-${i + 1}`}
+                                      icon={<MinusCircleIcon />}
+                                      onClick={() => {
+                                        removeParam(i);
+                                        paramArrayHelpers.remove(i);
+                                      }}
+                                    >
+                                      Remove parameter
+                                    </Button>
+                                  </DataListCell>,
+                                ]}
+                              />
+                            </DataListItemRow>
+                            <DataListContent
+                              hasNoPadding
+                              aria-label="list-item-content"
+                              isHidden={!expanded[i]}
+                            >
+                              <DataListItemRow className="pf-v5-u-pl-0">
+                                <DataListItemCells
+                                  dataListCells={[
+                                    <DataListCell
+                                      key="param-name"
+                                      width={3}
+                                      className="pf-v5-u-pl-xl pf-v5-u-pt-0"
+                                    >
+                                      <FormGroup label="Name" isRequired>
```

Comment:
Name and value field is optional in design doc.

```suggestion
                                      <FormGroup label="Name">
```
<img width="856" alt="image" src="https://github.com/openshift/hac-dev/assets/9964343/0905dcf9-2493-4ff3-a1c1-c963123f3196">